### DISCUSSION TITLE
Issue accessing static files in Next.js with dynamic routes in App Router

### DISCUSSION THREAD

**Discussion by JuanSpada** [vercel/next.js#67246] [App Router]
I'm working on a Next.js application and encountering an issue when trying to access static files while using dynamic routes in the App Router.

My project structure is as follows:

```
/public
  /images
    img.png
/app
  /[companyslug]
    page.jsx
```

When I try to access the static file at localhost:3000/images/img.png, I get an error indicating that the companyslug is not found, as if Next.js is trying to interpret the /images/img.png path as a dynamic route.

Is there a way to configure Next.js to properly serve the static files from the public directory without conflicting with the dynamic routes?

Thanks

**Comment by i-brad** [comment]
You should be able to access static files with no issues in dynamic routes

<img width="206" alt="image" src="https://github.com/vercel/next.js/assets/84320904/b6744091-3f9b-4143-ade9-4f7b5549afb8">

having something like this with the src attr value pointing to an image or SVG file within your public folder should work fine.

**Comment by JuanSpada** [comment]
Yes If I use next image works. But If I go to that URL directly instead of compiling it with Next.js Im having that issue. I need it to work also when I go to https://localhost:3000/images/img.png So I can use that img on my emails.

Thanks

**Comment by JuanSpada** [comment]
...............................

**Comment by axelmarciano** [comment]
Hi @JuanSpada,
Can you share your middleware.ts ?
